samples/scanner/lb.yaml

```diff
@@ -0,0 +1,32 @@
+rules:
```

Comment:
These are nice examples -- it helped me understand what you are trying to do.

Few comments,

* Could you make some of the values (e.g. name of the rule) a little more descriptive of the allowed LB rule?
* Also, I think the other data (e.g. IP addresses, LB target) look generic, but just want to make sure they really are.
* Could you also include the license header? (https://github.com/GoogleCloudPlatform/forseti-security/blob/master/samples/scanner/bigquery_rules.yaml#L1)
* Last but not least, and sorry for being so nitpicky... Would it be ok to rename this yaml file as something like "load_balancer.yaml"?